examples/mxnet/tune/mx_job_tune_gpu_v1beta1.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: "kubeflow.org/v1beta1"
```

Comment:
Do we need `v1beta` examples since we support only v1 version of API? 
cc @kubeflow/wg-training-leads

Reply:
This file should been deleted

Reply:
I can confirm we don't want to bring any non v1 examples into this repo